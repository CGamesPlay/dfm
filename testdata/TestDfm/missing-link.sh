#!/bin/bash
# Tests dfm correctly handling a file that has been unlinked behind dfm's back.
set -e
. "$(dirname "$0")/../helpers.sh"

export HOME="$(pwd)/home"
export DFM_DIR="$HOME/dfmdir"

mkdir -p ~/dfmdir/files
echo 'config' > ~/dfmdir/files/.bashrc

dfm init --repos files
dfm link

rm ~/.bashrc
rm ~/dfmdir/files/.bashrc
dfm link
dfm link # no output on second run
