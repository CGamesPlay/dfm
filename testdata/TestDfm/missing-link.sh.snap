$ dfm init --repos files
Initialized /test/home/dfmdir as a dfm directory.
$ dfm link
files/.bashrc -> /test/home/.bashrc
$ dfm link
removed .bashrc
$ dfm link
